Bottom: 01103d251621c7ec796ad03a7dd53cefd6b0ccea
Top:    ffae4082a320c8bdcae008646a9819480d567df9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 23:22:28 +0000

Fix rubberbanding not to touch rat-lines


---

diff --git a/src/rubberband.c b/src/rubberband.c
index aec6913..e54d61a 100644
--- a/src/rubberband.c
+++ b/src/rubberband.c
@@ -78,7 +78,7 @@ test_circle_hits_box (PointType *point, Coord radius, BoxType *box)
       box->Y1 - radius <= point->Y && point->Y <= box->Y2 + radius)
     {
       /* We are close enough to the box boundary that we might touch
-       * it, althoguh it is possible that if we are near one of the
+       * it, although it is possible that if we are near one of the
        * box corners, the circle contour doesn't touch the box.
        */
